package.json

```diff
@@ -84,9 +84,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@polkadot/api": "1.29.1",
-    "@polkadot/util": "3.2.1",
-    "@polkadot/util-crypto": "3.2.1",
+    "@polkadot/api": "2.3.1",
+    "@polkadot/util": "3.6.1",
+    "@polkadot/util-crypto": "3.6.1",
```

Comment:
Is there any breaking change from polkadot on these versions?

Reply:
Already addressed